Documentation update for SCP-B2B-1.0.19 and SCP-4.8.18 (#829)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/storefront-compatibility/v247.mdx

- [`generateCustomerToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/generate-token/) mutation - Creates a new customer token.

> - [`guestOrder`](https://developer.adobe.com/commerce/webapi/graphql/schema/orders/queries/guest-order/) query - Retrieves information about a guest order using an order number, email, and billing last name as input.

Reviewer: ```suggestion
- [`guestOrder`](https://developer.adobe.com/commerce/webapi/graphql/schema/orders/queries/guest-order/) query - Retrieves guest order information for the specified order number, email, and billing last name.
```

Author: Fixed

> - [`guestOrderByToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/orders/queries/guest-order-by-token/) query - Retrieves information about a guest order based on an order token.

Reviewer: ```suggestion
- [`guestOrderByToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/orders/queries/guest-order-by-token/) query - Retrieves guest order data associated with a given order token.
```

Author: Fixed
